tests/test_services.py

```
@@ -116,3 +169,40 @@ async def test_set_sync_state_exception(hass: HomeAssistant, mock_api: Mock) ->
         intensity=None,
         hue_target=None,
     )
+
+async def test_set_sync_state_retry_on_invalid_state_streaming(
+    hass: HomeAssistant,
+    mock_api: Mock,
+) -> None:
+    mock_api.hue.groups[1]._raw["active"] = True  # noqa: SLF001
```

Ruff (0.14.10)

Unused `noqa` directive (non-enabled: `SLF001`)

Remove unused `noqa` directive

(RUF100)